### ISSUE TITLE
On the server, some options of the settings panel are not displayed

### ISSUE THREAD

**Issue by cary-rowen** [leonardder/rdAccess#8]
As the title says, I use mstsc to connect to the remote desktop, open the NVDA settings panel located on the server, and I find that the options in the 'server' group under the 'Remote Desktop' category are not displayed correctly.

**Comment by cary-rowen** [comment]
I tested version 0.5 and '403a1faa31a9cdb13cc8935e7eecf2e1ee999115' can reproduce.

**Comment by leonardder** [comment]
Could you clarify what you're missing?

**Comment by cary-rowen** [comment]
Sorry, I should have checked if only "Enable NVDA RD for" can be adjusted by the user on the server, which is the case on my side.
Both grouped controls "Server" and "Client" are invisible to me.b

**Comment by leonardder** [comment]
Are you getting any errors in the log when opening the panel or toggling the server option?

**Comment by cary-rowen** [comment]
There is no error in the log, but I can't focus on 'Automatically recover remote speech after connection loss', but I can see it in the log.

Here is the log snippet:
```
IO - speech.speech.speak (12:24:44.483) - MainThread (8176):
Speaking ['日志片段开始点已标记，再按一次复制到剪贴板']
IO - inputCore.InputManager.executeGesture (12:24:44.955) - winInputHook (7684):
Input: kb(laptop):upArrow
DEBUG - gui.contextHelp.bindHelpEvent (12:24:44.968) - MainThread (8176):
Did context help binding for RemoteDesktopSettingsPanel
DEBUG - gui.contextHelp.bindHelpEvent (12:24:44.994) - MainThread (8176):
Did context help binding for AutoWidthColumnListCtrl
IO - speech.speech.speak (12:24:45.075) - MainThread (8176):
Speaking ['Remote Desktop', CancellableSpeech (still valid)]
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._getIA2RelationFirstTarget (12:24:45.077) - MainThread (8176):
Unable to use _getIA2TargetsForRelationsOfType, fallback to _IA2Relations.
DEBUG - NVDAObjects.IAccessible.IAccessible._get__IA2Relations (12:24:45.077) - MainThread (8176):
Not an IA2.IAccessible2
DEBUG - NVDAObjects.IAccessible.IAccessible._getIA2RelationFirstTarget (12:24:45.077) - MainThread (8176):
Unable to fetch _IA2Relations
Traceback (most recent call last):
  File "NVDAObjects\IAccessible\__init__.pyc", line 1600, in _getIA2RelationFirstTarget
  File "baseObject.pyc", line 62, in __get__
  File "baseObject.pyc", line 168, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 1515, in _get__IA2Relations
NotImplementedError
DEBUG - external:addons.rdAccess.lib.protocol.CommandHandlerStore.__call__ (12:24:45.247) - remoteSynthDriver_2 (6820):
Getting handler on <addons.rdAccess.lib.protocol.CommandHandlerStore object at 0x037A7C70> to process command 120
DEBUG - external:addons.rdAccess.lib.protocol.CommandHandler.__call__ (12:24:45.247) - remoteSynthDriver_2 (6820):
Calling <addons.rdAccess.lib.protocol.CommandHandler object at 0x0520F7F0> for command <SpeechCommand.INDEX_REACHED: 120>
IO - inputCore.InputManager.executeGesture (12:24:45.834) - winInputHook (7684):
Input: kb(laptop):tab
IO - speech.speech.speak (12:24:45.957) - MainThread (8176):
Speaking ['Remote Desktop', '属性页', CancellableSpeech (still valid)]
IO - speech.speech.speak (12:24:45.959) - MainThread (8176):
Speaking ['Enable NVDA RD for', '列表', CancellableSpeech (still valid)]
IO - speech.speech.speak (12:24:45.961) - MainThread (8176):
Speaking ['Incoming connections (Remote Desktop Server)', '复选框', '已选中', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (12:24:46.430) - winInputHook (7684):
Input: kb(laptop):tab
IO - speech.speech.speak (12:24:46.467) - MainThread (8176):
Speaking ['Server', '分组', CancellableSpeech (still valid)]
IO - speech.speech.speak (12:24:46.469) - MainThread (8176):
Speaking ['Automatically recover remote speech after connection loss', '复选框', '已选中', 'Alt+', CharacterModeCommand(True), 'a', CharacterModeCommand(False), CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (12:24:47.707) - winInputHook (7684):
Input: kb(laptop):tab
IO - speech.speech.speak (12:24:47.720) - MainThread (8176):
Speaking ['确认', '按钮', CancellableSpeech (still valid)]
DEBUG - external:addons.rdAccess.lib.protocol.CommandHandlerStore.__call__ (12:24:47.825) - remoteSynthDriver_1 (6056):
Getting handler on <addons.rdAccess.lib.protocol.CommandHandlerStore object at 0x037A7C70> to process command 120
DEBUG - external:addons.rdAccess.lib.protocol.CommandHandler.__call__ (12:24:47.825) - remoteSynthDriver_1 (6056):
Calling <addons.rdAccess.lib.protocol.CommandHandler object at 0x0520F7F0> for command <SpeechCommand.INDEX_REACHED: 120>
IO - inputCore.InputManager.executeGesture (12:24:49.282) - winInputHook (7684):
Input: kb(laptop):shift+control+NVDA+f1
```

**Comment by leonardder** [comment]
So if I understand you correctly, your system only announces "server grouping" and doesn't announce the option even though it is in the log that it did announce it.
Could you please try [this](https://github.com/leonardder/rdAccess/releases/download/latest/rdAccess-81197f7ea792c6d1091d246e930643a46b55f9d2.nvda-addon). I added some fixes in that build regarding speech.
Could you also have a try with espeak on your local system?

**Comment by leonardder** [comment]
I will have a look at world voice

**Comment by leonardder** [comment]
World voice seems to be a very complex thing. It's probably best to report this on their end, but other than that, there's not much I can do.
